airflow/contrib/operators/ssh_execute_operator.py

```diff
@@ -55,7 +55,7 @@ def __enter__(self):
         prefix = self._prefix
 
         pmktemp = ssh_hook.Popen(["-q",
-                                  "mktemp", "--tmpdir", prefix + "_XXXXXX"],
+                                  "mktemp", "-t", prefix + "_XXXXXX"],
```

Comment:
something like https://github.com/apache/incubator-airflow/compare/master...zodiac:xuanji/mktemp_fix but making sure that the envar expansion happens on the remote machine